lib/files.test.js

```diff
@@ -164,12 +165,51 @@ describe(_.startCase(filename), function () {
 
     it('returns result', function () {
       var filename = 'some-name',
+        result = 'some result',
+        expected = 'some result';
+
+      yaml.safeLoad.returns(temp2env(result));
+
+      expect(fn(filename)).to.equal(expected);
+    });
+
+    it('returns result from first file', function () {
+      var filename = 'some-name',
+        result = 'some result';
+
+      fs.readFileSync.onCall(0).returns(result);
+      fs.readFileSync.onCall(1).throws();
+      yaml.safeLoad.returnsArg(0);
+
+      expect(fn(filename)).to.equal(result);
+    });
+
+    it('returns result from second file', function () {
+      var filename = 'some-name',
         result = 'some result';
 
-      yaml.safeLoad.returns(result);
+      fs.readFileSync.onCall(0).throws();
+      fs.readFileSync.onCall(1).returns(result);
+      yaml.safeLoad.returnsArg(0);
 
       expect(fn(filename)).to.equal(result);
     });
+  });
+
+  describe('getYamlWithEnv', function () {
+    var fn = lib[this.title];
+
+    it('returns result', function () {
+      var filename = 'some-name',
+        result = 'some ${FOO}',
+        expected = 'some result';
+
+      process.env.FOO = 'result';
+
+      yaml.safeLoad.returns(temp2env(result));
+
+      expect(fn(filename)).to.equal(expected);
+    });
```

Comment:
Again, it seems like you just copy and pasted nearby code.  That's very scary.

If you're not adding new logic that needs to be tested, then you're probably not doing something original.  More thought needs to be put into this.